### PR TITLE
knative 0.9 vendor changes

### PR DESCRIPTION
**Pull request by animeshsingh**
knative 0.9 vendor changes

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/523)

### CODE REVIEW

**Comment by cliveseldon** [comment]
@animeshsingh Does this accidentially delete cloudevents again?

**Comment by cliveseldon** [comment]
And opencensus? Not sure why this is happening if you are going from master?

**Comment by animeshsingh** [comment]
@cliveseldon that's the only reason I can think of - its pruning packages not used by the code. Maybe worthwhile bringing it just before the logger code is goes in?

**Comment by cliveseldon** [comment]
@animeshsingh Simpler I think is #503 gets merged and then you can rebase and it won't delete these.
@ellis-bigelow I've done changes to #503 so can you check it its ready to merge?

File: Gopkg.toml [hunk 1]
```diff
@@ -46,7 +46,7 @@ version="v1.4.7"
 
 [[constraint]]
   name = "knative.dev/serving"
-  version = "=v0.8.0"
```
Comment (yuzisun) [inline Gopkg.toml]: Let's make this change after we release 0.2.1 ?  A bunch of fixes need to get into that and manifests are still on knative 0.8.

Comment (animeshsingh) [reply Gopkg.toml]: @yuzisun what are we waiting for in 0.2.1?

**Pull request closed without merging**